src/actions/fetch/copier.ts

```diff
@@ -72,6 +71,18 @@ export class CopierRunner {
     await fs.ensureDir(intermediateDir);
     const resultDir = path.join(workspacePath, 'result');
 
+    // First lets grab the default copier.json file
+    const copierJson = await this.fetchTemplateCopier(
+      templateContentsDir,
+    );
+
+    const copierInfo = {
+      ...copierJson,
+      ...values,
+    };
+
+    await fs.writeJSON(path.join(templateDir, 'copier.json'), copierInfo);
+
```

Comment:
I don't understand what the `copier.json` is there for. Looks like a leftover from the cookiecutter on that made a json file in the root... Can this be deleted?

Reply:
I thought this was the way to read the template.yaml (or in our case the catalog-info.yaml) file to see what variables are needed. Not completely sure though. Once I'm back working on the containers part (after this EPICS training week, if that's okay), I'll try deleting this part of the code and test to see if it still works / is needed

Reply:
okay yeah this is not needed
plugin works as required without this section
deleting this section even allows me to simplify other sections too to suit our usages